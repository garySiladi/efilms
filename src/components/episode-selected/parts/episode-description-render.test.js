// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeDescription from './episode-description';

test('EpisodeDescription renders correctly', () => {
  const tree : string = renderer.create(<EpisodeDescription />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('EpisodeDescription renders with data', () => {
  const tree : string = renderer.create(<EpisodeDescription
    title="title xyz"
    subtitle="subtitle xyz"
    description="description xyz"
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
