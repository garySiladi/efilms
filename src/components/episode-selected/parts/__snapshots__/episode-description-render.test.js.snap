// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodeDescription renders correctly 1`] = `
<div
  className="episode-description"
>
  <h3
    className="episode-description__subtitle"
  />
  <h2
    className="episode-description__title"
  />
  <p
    className="episode-description__text"
  />
</div>
`;

exports[`EpisodeDescription renders with data 1`] = `
<div
  className="episode-description"
>
  <h3
    className="episode-description__subtitle"
  >
    subtitle xyz
  </h3>
  <h2
    className="episode-description__title"
  >
    title xyz
  </h2>
  <p
    className="episode-description__text"
  >
    description xyz
  </p>
</div>
`;
